src/FastExpressionCompiler.LightExpression/Expression.cs

```diff
@@ -25,7 +25,6 @@ THE SOFTWARE.
 
 // ReSharper disable CoVariantArrayConversion
 #nullable disable
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member

```

Comment:
I removed these as they don't seem to matter for your development. Must have been something added due to internal files.